pkg/karmadactl/cmdinit/kubernetes/deploy.go

```diff
@@ -310,11 +286,22 @@ func (i *InstallOptions) initComponent() {
 	if err := i.CreateService(i.karmadaWebhookService()); err != nil {
 		klog.Exitln(err)
 	}
-
 	if _, err := i.KubeClientSet.AppsV1().Deployments(i.Namespace).Create(context.TODO(), i.makeKarmadaWebhookDeployment(), metav1.CreateOptions{}); err != nil {
 		klog.Warning(err)
 	}
-	if err := WaitPodReady(i.KubeClientSet, i.Namespace, utils.MapToString(webhookLabels), 30); err != nil {
+	if err := WaitPodReady(i.KubeClientSet, i.Namespace, utils.MapToString(webhookLabels), waitPodReadyTimeout); err != nil {
+		klog.Warning(err)
+	}
+	// Create karmada-aggregated-apiserver
+	// https://github.com/prodanlabs/karmada/blob/master/artifacts/deploy/karmada-aggregated-apiserver.yaml
```

Comment:
ditto

Reply:
The copied URL address is wrong. I'll change it.